Serbian/Macedonian Cyrillic shapes

Author: W. Kerr <w@kerr.sh>
Modifies:
 U+0431 CYRILLIC SMALL LETTER BE
 U+0433 CYRILLIC SMALL LETTER GHE
 U+0434 CYRILLIC SMALL LETTER DE
 U+043F CYRILLIC SMALL LETTER PE
 U+0442 CYRILLIC SMALL LETTER TE
--- a/leggie-12.bdf
+++ b/leggie-12.bdf
@@ -12285,11 +12285,11 @@
 BBX 6 12 0 -2
 BITMAP
 00
-04
-18
-20
+00
+38
 40
-78
+20
+38
 44
 44
 44
--- a/leggie-18.bdf
+++ b/leggie-18.bdf
@@ -16157,12 +16157,12 @@
 0000
 0000
 0000
-0100
-1E00
-2000
+3E00
 4000
 4000
-7E00
+2000
+1000
+3E00
 4100
 4100
 4100
--- a/leggie-24.bdf
+++ b/leggie-24.bdf
@@ -20027,15 +20027,15 @@
 0000
 0000
 0000
-0060
-07C0
+0000
 1F80
-1800
-3000
-3000
+3F80
+6000
 6000
-7F80
-7FC0
+3800
+0E00
+1F80
+3FC0
 6060
 6060
 6060
--- a/leggie-12b.bdf
+++ b/leggie-12b.bdf
@@ -8314,11 +8314,11 @@
 BBX 6 12 0 -2
 BITMAP
 00
-0C
-38
-60
+00
+78
 C0
-F8
+60
+78
 CC
 CC
 CC
--- a/leggie-12bi.bdf
+++ b/leggie-12bi.bdf
@@ -8314,11 +8314,11 @@
 BBX 6 12 0 -2
 BITMAP
 00
-04
-18
-30
+00
+1C
 30
-78
+10
+38
 6C
 CC
 D8
@@ -8353,14 +8353,14 @@
 BITMAP
 00
 00
+38
 00
 00
-00
-78
-0C
-78
-C0
-78
+30
+30
+60
+60
+30
 00
 00
 ENDCHAR
@@ -8372,16 +8372,16 @@
 BITMAP
 00
 00
+00
+00
+00
 38
-0C
-0C
-3C
 6C
 CC
 D8
+78
+18
 70
-00
-00
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -8581,14 +8581,14 @@
 BITMAP
 00
 00
+3C
 00
 00
-00
-78
+6C
 6C
 CC
 D8
-D8
+78
 00
 00
 ENDCHAR
@@ -8638,14 +8638,14 @@
 BITMAP
 00
 00
+7C
 00
 00
-00
-68
-7C
+54
+54
 FC
 F8
-D8
+78
 00
 00
 ENDCHAR
--- a/leggie-12i.bdf
+++ b/leggie-12i.bdf
@@ -8314,11 +8314,11 @@
 BBX 6 12 0 -2
 BITMAP
 00
-04
-18
-20
+00
+1C
 20
-78
+10
+38
 44
 84
 88
@@ -8353,14 +8353,14 @@
 BITMAP
 00
 00
+1C
 00
 00
-00
-78
-04
-78
-80
-78
+10
+10
+20
+20
+10
 00
 00
 ENDCHAR
@@ -8372,16 +8372,16 @@
 BITMAP
 00
 00
+00
+00
+00
 38
-04
-04
-3C
 44
 84
 88
+78
+08
 70
-00
-00
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -8581,14 +8581,14 @@
 BITMAP
 00
 00
+3C
 00
 00
-00
-78
+44
 44
 84
 88
-88
+78
 00
 00
 ENDCHAR
@@ -8638,14 +8638,14 @@
 BITMAP
 00
 00
+7C
 00
 00
-00
-68
 54
-A4
+54
+94
 A8
-88
+78
 00
 00
 ENDCHAR
--- a/leggie-18b.bdf
+++ b/leggie-18b.bdf
@@ -10932,12 +10932,12 @@
 0000
 0000
 0000
-0300
-1E00
-3000
+3F00
 6000
 6000
-7F00
+3000
+1800
+3F00
 6180
 6180
 6180
--- a/leggie-18bi.bdf
+++ b/leggie-18bi.bdf
@@ -10932,12 +10932,12 @@
 0000
 0000
 0000
-0080
-0F00
-1800
+1F80
 3000
-6000
-7F00
+3000
+1800
+1800
+3F00
 6180
 6180
 C180
@@ -10984,16 +10984,16 @@
 0000
 0000
 0000
+1F00
 0000
 0000
-0000
-7F00
-0180
-0180
-7F00
-C000
-C000
-7F00
+0C00
+0C00
+0C00
+1800
+1800
+1800
+0C00
 0000
 0000
 0000
@@ -11007,21 +11007,21 @@
 0000
 0000
 0000
-1F00
-0180
-0180
-0180
-0180
-3F00
-6300
-6300
-C300
-C600
-C600
-7C00
 0000
 0000
 0000
+0000
+0000
+3F00
+6180
+6180
+C180
+C300
+C300
+7F00
+0300
+0600
+7C00
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -11284,16 +11284,16 @@
 0000
 0000
 0000
+3F80
 0000
 0000
-0000
-7F00
+6180
 6180
 6180
 C180
 C300
 C300
-C300
+7F00
 0000
 0000
 0000
@@ -11359,16 +11359,16 @@
 0000
 0000
 0000
+3F80
 0000
 0000
-0000
-7B00
+6D80
 6D80
 6D80
 CD80
-D980
 DB00
-C300
+DB00
+FF00
 0000
 0000
 0000
--- a/leggie-18i.bdf
+++ b/leggie-18i.bdf
@@ -10932,12 +10932,12 @@
 0000
 0000
 0000
-0080
-0F00
-1000
+1F00
 2000
-4000
-7E00
+2000
+1000
+1000
+3E00
 4100
 4100
 8100
@@ -10984,16 +10984,16 @@
 0000
 0000
 0000
+1E00
 0000
 0000
-0000
-7E00
-0100
-0100
-7E00
-8000
-8000
-7E00
+0800
+0800
+0800
+1000
+1000
+1000
+0800
 0000
 0000
 0000
@@ -11007,21 +11007,21 @@
 0000
 0000
 0000
-1F00
-0080
-0080
-0080
-0080
-3F00
+0000
+0000
+0000
+0000
+0000
+3E00
 4100
 4100
 8100
 8200
 8200
-7C00
-0000
-0000
-0000
+7E00
+0200
+0400
+7800
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -11284,16 +11284,16 @@
 0000
 0000
 0000
+3F00
 0000
 0000
-0000
-7E00
+4100
 4100
 4100
 8100
 8200
 8200
-8200
+7E00
 0000
 0000
 0000
@@ -11359,16 +11359,16 @@
 0000
 0000
 0000
+3F00
 0000
 0000
-0000
-7600
+4900
 4900
 4900
 8900
-9100
 9200
-8200
+9200
+FE00
 0000
 0000
 0000
--- a/leggie-24b.bdf
+++ b/leggie-24b.bdf
@@ -13548,15 +13548,15 @@
 0000
 0000
 0000
-0070
-07E0
+0000
 1FC0
-1C00
-3800
-3800
+3FC0
+7000
 7000
-7FC0
-7FE0
+3C00
+0F00
+1FC0
+3FE0
 7070
 7070
 7070
--- a/leggie-24bi.bdf
+++ b/leggie-24bi.bdf
@@ -13548,15 +13548,15 @@
 0000
 0000
 0000
-0030
-01F0
-07E0
-0F00
+0000
+07F0
+0FF0
 1C00
-3800
-3800
-7FC0
-7FE0
+1C00
+0E00
+0780
+1FC0
+3FE0
 7070
 7070
 E070
@@ -13613,20 +13613,20 @@
 0000
 0000
 0000
+0FC0
+0FC0
 0000
 0000
-0000
-0000
-1FC0
-1FE0
-0070
-0070
-3FE0
-7FC0
-E000
-E000
-7F80
-3F80
+0700
+0700
+0700
+0700
+0E00
+0E00
+0E00
+0E00
+0E00
+0700
 0000
 0000
 0000
@@ -13640,28 +13640,28 @@
 BITMAP
 0000
 0000
-0F00
-0FC0
-01E0
-0070
-0070
-0070
-0070
-0070
-1FF0
-3FE0
-70E0
-70E0
-E0E0
-E0E0
-E1C0
-E1C0
-7F80
-3F00
 0000
 0000
 0000
 0000
+0000
+0000
+0000
+0000
+1FE0
+3FF0
+7070
+7070
+E070
+E070
+E0E0
+E0E0
+7FE0
+3FE0
+00E0
+01C0
+3FC0
+3F00
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -13985,20 +13985,20 @@
 0000
 0000
 0000
+1FE0
+1FE0
 0000
 0000
-0000
-0000
-7FC0
-7FE0
+7070
+7070
 7070
 7070
 E070
 E070
 E0E0
 E0E0
-E0E0
-E0E0
+7FE0
+3FE0
 0000
 0000
 0000
@@ -14078,20 +14078,20 @@
 0000
 0000
 0000
+3FE0
+3FE0
 0000
 0000
-0000
-0000
-7DC0
-7FE0
+7770
+7770
 7770
 7770
 E770
 EE70
 EEE0
 EEE0
-E0E0
-E0E0
+7FE0
+3BE0
 0000
 0000
 0000
--- a/leggie-24i.bdf
+++ b/leggie-24i.bdf
@@ -13548,15 +13548,15 @@
 0000
 0000
 0000
-0030
-01E0
-07C0
-0E00
+0000
+07E0
+0FE0
 1800
-3000
-3000
-7F80
-7FC0
+1800
+0C00
+0700
+1F80
+3FC0
 6060
 6060
 C060
@@ -13613,20 +13613,20 @@
 0000
 0000
 0000
+0FC0
+0FC0
 0000
 0000
-0000
-0000
-1F80
-1FC0
-0060
-0060
-3FC0
-7F80
-C000
-C000
-7F00
-3F00
+0600
+0600
+0600
+0600
+0C00
+0C00
+0C00
+0C00
+0C00
+0600
 0000
 0000
 0000
@@ -13640,28 +13640,28 @@
 BITMAP
 0000
 0000
-0700
-07C0
-00E0
-0030
-0030
-0030
-0030
-0030
-1FF0
-3FF0
+0000
+0000
+0000
+0000
+0000
+0000
+0000
+0000
+1FC0
+3FE0
 6060
 6060
 C060
 C060
 C0C0
 C0C0
-7F80
-3F00
-0000
-0000
-0000
-0000
+7FC0
+3FC0
+00C0
+0180
+3F80
+3E00
 ENDCHAR
 STARTCHAR CYRILLIC SMALL LETTER IE
 ENCODING 1077
@@ -13985,20 +13985,20 @@
 0000
 0000
 0000
+1FC0
+1FC0
 0000
 0000
-0000
-0000
-7F80
-7FC0
+6060
+6060
 6060
 6060
 C060
 C060
 C0C0
 C0C0
-C0C0
-C0C0
+7FC0
+3FC0
 0000
 0000
 0000
@@ -14078,20 +14078,20 @@
 0000
 0000
 0000
+1FC0
+1FC0
 0000
 0000
-0000
-0000
-7980
-7FC0
+6660
+6660
 6660
 6660
 C660
 CC60
 CCC0
 CCC0
-C0C0
-C0C0
+7FC0
+33C0
 0000
 0000
 0000
